src/sentry/static/sentry/app/utils/discover/fields.tsx

```diff
@@ -222,20 +222,16 @@ export const AGGREGATIONS = {
   },
 } as const;
 
-assert(
-  AGGREGATIONS as Readonly<
-    {
-      [key in keyof typeof AGGREGATIONS]: {
-        parameters: Readonly<AggregateParameter[]>;
-        // null means to inherit from the column.
-        outputType: null | ColumnType;
-        isSortable: boolean;
-      };
-    }
-  >
-);
+assert(AGGREGATIONS as Readonly<{[key in keyof typeof AGGREGATIONS]: Aggregation}>);
+
+export type AggregationKey = keyof typeof AGGREGATIONS | '';
```

Comment:
`AggregateKey` makes more sense for the names, and `Aggregation` makes more sense as a config IMO.